.github/workflows/ci.yml

```diff
@@ -7,13 +7,15 @@ on:
   pull_request:
     branches:
       - master
+  schedule:
+    - cron: '00 12 * * 1' # Runs every Monday at 8:00 AM EST
 
 jobs:
   Test:
     runs-on: ubuntu-latest
```

Comment:
Could we add windows and mac-os as in the other libs?

Reply:
Done. Included the windows and mac-os in the CI job